services/movie.py

```diff
@@ -6,22 +7,26 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None
```

Comment:
The `title` parameter should be a single string, not a list of strings, as it is used to filter movies by a single title. Change the type hint to `title: str = None`.